### PR TITLE
ENG-2920

### PR DESCRIPTION
**Pull request by fjlanasa**
JIRA: https://etmainc.atlassian.net/browse/ENG-2920

Small changes to app config and `Price` model to support changes to Rails app in: https://github.com/ETMAInc/TruffleHunt-WhitelabelSite/pull/864

### CODE REVIEW

File: src/search/product/models/index.ts [hunk 1]
```diff
@@ -100,6 +103,10 @@ export default class Product implements IProduct {
     return this.isBookable;
   }
 
+  getIsDeepLink(): boolean {
```
Comment (brianeroth) [inline src/search/product/models/index.ts]: Is `isDeepLink()` a better name for this?

File: src/search/product/models/price.ts [hunk 2]
```diff
@@ -8,23 +8,28 @@ export interface IPrice {
...
+    i: number,
+    currencyCode = Currencies.USD.code,
+    currencyFormatOpts = {
       format: "%s%v %c",
-      code: this.currency.code,
-      symbol: this.currency.symbol_native,
-      minFraction: 0,
-      maxFraction: this.currency.decimal_digits
-    };
+      code: Currencies.USD.code,
```
Comment (brianeroth) [inline src/search/product/models/price.ts]: Isn't this going to ignore the currency code that we've passed in? It'll always use USD?

File: src/search/product/models/index.ts (hunk 1 above)
Comment (Sodman) [reply src/search/product/models/index.ts]: ^Agree in general - but we also already have getIsBookable() and getTriedAndTrue(), so whatever we end up with we should ideally be consistent.

Comment (fjlanasa) [reply src/search/product/models/index.ts]: I'll go with `isDeepLink` and `isBookable`

Comment (fjlanasa) [reply src/search/product/models/index.ts]: went with `isDeepLink` and `isBookable` for function names, but then had to change attributes to `deepLink` and `bookable`

**Pull request merged**